src/index.js

```diff
@@ -29,15 +29,29 @@ if (
 }
 
 // Event JSON example: https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#webhook-payload-example-32
-const eventJSON = JSON.parse(
-  readFileSync(process.env.GITHUB_EVENT_PATH, "utf8")
-);
+let eventJSON = JSON.parse(readFileSync(process.env.GITHUB_EVENT_PATH, "utf8"));
 
 // Examples of event types: "workflow_dispatch", "push", "pull_request", etc
 const eventType = process.env.GITHUB_EVENT_NAME;
 
```

Comment:
We should add a check roundabout here that confirms the eventType is supported. If not, then we print a good error message an exit. Something like this:
```
if (eventType not in ["workflow_dispatch", "push", "pull_request", "schedule"] {
    "ERROR ${eventType} is not currently supported by Gitleaks-Action."
    exit(1);
}

Reply:
We already have a similar check in `start()` but I agree that we should just check it here

Reply:
Ah nice!